src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -3,101 +3,95 @@ title: Learning Dart as a JavaScript developer
 description: Leverage your JavaScript knowledge when learning Dart.
 ---
 
-This guide aims to leverage your JavaScript (JS)
-programming knowledge when learning Dart.
-It showcases key similarities and differences
-in both languages, and introduces Dart concepts
-that are not present in vanilla JavaScript.
-As a JavaScript developer,
-Dart should feel quite familiar,
-as both languages share many concepts. 
+This guide aims to leverage your JavaScript programming knowledge
+when learning Dart.
+It showcases key similarities and differences in both languages,
+and introduces Dart concepts that unsupported in JavaScript.
+As a JavaScript developer, Dart should feel quite familiar,
+as both languages share many concepts.
 
 Like JavaScript, Dart runs on an event loop,
 so both languages execute code in a similar way.
 For example, asynchronous concepts like futures
-(promises in JS) and the `async/await` syntax are very similar.
+(promises in JavaScript) and the `async/await` syntax are very similar.
 
 Dart is strongly typed, unlike JavaScript.
-If you are also familiar with TypeScript or tooling
-that adds static typing (like Flow), then learning Dart
-probably won't be much of an extra step for you.
-However, if you've mostly worked with vanilla JavaScript,
-it might take a bit more effort to adjust.
-The good news is that since Dart is strongly typed,
-many errors that might exist in JavaScript code are
-caught even before compiling your Dart code.
-
-As of Dart 2.12, null safety is enabled by default,
-which JavaScript doesn't support. As a JavaScript developer,
+If you have used with TypeScript or Flow,
+this should simplify learning Dart.
+If you've mostly worked with pure JavaScript,
+it might be more of an adjustment.
+With strong typing, Dart catches many errors before compiling
+that might exist in JavaScript code.
+
+Dart enables null safety by default.
+JavaScript doesn't support null safety.
+As a JavaScript developer,
 it might take a while to learn how to write null safe code,
 but the trade-off is better protection against
 null reference exceptions that are detected even
 before compiling Dart code. (Thereby avoiding those
 dreaded `TypeError`s that occur when doing operations
-on a JS variable that turns out to be null.)
+on a JavaScript variable that turns out to be null.)
 
 ## Conventions and linting
 
-JavaScript and Dart both have linting tools
-to enforce standard conventions. However,
-while JavaScript has `ESLint` as a standalone tool
-(among other available tooling),
-and many different standards and configurations,
-Dart has official layout conventions and includes
-a linter to make compliance effortless.
+JavaScript and Dart both have linting tools to enforce standard conventions.
+While JavaScript offers many tools, standards, and configurations,
+Dart has one official layout conventions and includes a linter to make
+compliance effortless. Dart calls this linter the analyzer.
```

Comment:
The analyzer is a lot more than just the linter, providing warnings and errors for misuse of language features,
code completion, actions like "Find usages", "Go to definition", "Fix X", etc.

```suggestion
Dart has official layout and styles conventions and includes
a linter to make compliance effortless. 
Support for this and other analysis functionality is provided by the analyzer.
```